docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -44,6 +44,14 @@ User-defined dependencies will be appended to the generated unit file, but any e
 
 Set the systemd unit name prefix for containers. The default is *container*.
 
+#### **--env**, **-e**=*env*
+
+Set environment variables to the systemd unit files.
+
+If an environment variable is specified without a value, Podman will check the host environment for a value and set the variable only if it is set on the host. As a special case, if an environment variable ending in __*__ is specified without a value, Podman will search the host environment for variables starting with the prefix and will add those variables to the systemd unit files.
```

Comment:
I see no evidence in the code supporting the described behavior.  Can you point me to the code?  A test or two exercising the behavior would be great.

Reply:
This behavior is realized by `envLib.ParseSlice` function.
https://github.com/containers/podman/blob/cafa12d17ae2786b30042cdee09a0e1841304162/cmd/podman/generate/systemd.go#L153

`envLib.ParseSlice` calls `env.parseEnv` function and parses environment variables.
https://github.com/containers/podman/blob/cafa12d17ae2786b30042cdee09a0e1841304162/pkg/env/env.go#L109-L123

And I add e2e tests that can verify these behaviors.

Reply:
Thanks, I wasn't aware of that yet.